internal/store/process_measurements.go

```diff
@@ -19,22 +19,26 @@ import (
 	"fmt"
 
 	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	om "github.com/mongodb/go-client-mongodb-ops-manager/opsmngr"
 	"github.com/mongodb/mongocli/internal/config"
 )
 
 type ProcessMeasurementLister interface {
-	ListProcessMeasurements(string, string, int, *atlas.ProcessMeasurementListOptions) (*atlas.ProcessMeasurements, error)
+	ListProcessMeasurements(string, string, *int, *atlas.ProcessMeasurementListOptions) (*atlas.ProcessMeasurements, error)
 }
 
 type ProcessMeasurementStore interface {
 	ProcessMeasurementLister
 }
 
 // GetAllProjects encapsulate the logic to manage different cloud providers
-func (s *Store) ListProcessMeasurements(groupID, host string, port int, opts *atlas.ProcessMeasurementListOptions) (*atlas.ProcessMeasurements, error) {
+func (s *Store) ListProcessMeasurements(groupID, host string, port *int, opts *atlas.ProcessMeasurementListOptions) (*atlas.ProcessMeasurements, error) {
```

Comment:
I have to be honest not a fan of this change and makes me miss method overloading from java,
Passing a nil can be confusing for users so I rather have two methods here
Naming up to you but some options
`AtlasProcessMeasurements` vs `OpsManagerProcessMeasurements`
Or `ProcessMeasurementsByHostnamePort` vs `ProcessMeasurementsByHostId`

Reply:
You could also delete `ProcessMeasurementStore` as I don't see us using both methods in a command and rather it go away than be a confusing interface with two unrelated methods